rfcs/0015-release-manager.md

```diff
@@ -0,0 +1,79 @@
+---
+feature: release-manager-nixos
+start-date: 2017-07-18
+author: Robin Gloster (@globin)
+co-authors: Franz Pletz (@fpletz)
+related-issues: --
+---
+
+# Summary
+[summary]: #summary
+
+NixOS currently has no process for electing release managers (RMs). We propose to
+switch to a model with two RMs, where each RM SHOULD
+serve for a consecutive term of two releases. A new RM is appointed
+by the previous team for each new release.
+
+# Motivation
+[motivation]: #motivation
+
+Currently release managing in NixOS has mostly been done by individuals who
+volunteered and were then chosen by the last release manager. Over the last
+few releases a process has been established and
+[documented](https://nixos.org/nixos/manual/index.html#release-process).
+As this makes it easier to cut a release this role should be passed on
+regularly and not be held by a single individual over a longer time.
```

Comment:
Aside from influence you also make sure that knowledge spreads. If one person keeps a role for a long time, they may not document it as well because its not relevant for others. Furthermore, it may improve connectivity with the community if it is seen that such role isn't limited to "just a few".

These are some potential arguments.